packages/data/addon/adapters/bard-facts.js

```diff
@@ -174,6 +172,27 @@ export default EmberObject.extend({
     }
   },
 
+  /**
+   * Gets host by name in [{name: String, uri: String}]
+   * config datastructure in config.navi.dataSources.
+   *
+   * @private
+   * @param {String} name - name of host to get
+   * @returns {String} - uri of fact datasource to use
+   */
+  _getHost(name) {
+    if (name) {
+      const host = config.navi.dataSources.find(dataSource => dataSource.name === name);
+      if (host && host.uri) {
+        return host.uri;
+      }
+      warn(`Fact host for ${name} requested but none was found in configuration. Falling back to default`, {
```

Comment:
It might be worth adding a test case to make sure the default is used when a non-existent host is passed in to this function.